src/seedu/addressbook/data/person/Person.java

```diff
@@ -70,6 +70,16 @@ public void setTags(Set<Tag> replacement) {
         tags.addAll(replacement);
     }
 
+    @Override
+    public int compareTo(Object other) {
```

Comment:
After type-casting Comparable, compareTo() takes in Person parameter, rather than Object parameter.